tests/_test_utils.py

```diff
@@ -193,29 +195,23 @@ def _save_or_check(c, tmp_path, circuit_name, save_figs, recursive=False, compac
         assert ok, msg
 
 
-if __name__ == "__main__":
```

Comment:
The tests under ` main` here are testing some of the functions written inside `_test_utils`. Could you create a new test file `test_test_utils.py` and move them there ?